charts/tenant-ui/values-e79518-dev.yaml

```diff
@@ -24,8 +24,8 @@ tenant_ui:
   smtp:
     server: apps.smtp.gov.bc.ca
     port: 25
-    senderAddress: DoNotReplyTractionPR@gov.bc.ca
-    innkeeperInbox: lucas.oneil@gov.bc.ca
+    senderAddress: DoNotReplyTractionDTS@gov.bc.ca
+    innkeeperInbox: emiliano.sune@quartech.com
```

Comment:
@esune think this is the values file for the DTS deployment? Is it used or did we go another way?

Reply:
It is used, but will get decommissioned soon (hopefully). Proposed new values are good 👍🏻